frontend/src/component/admin/auth/SsoGroupSettings.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import { FormControlLabel, Grid, Switch, TextField } from '@mui/material';
+
+interface SsoGroupSettingsProps {
+    ssoType: string;
```

Comment:
We could be a bit stricter here if we wanted to, could also be an enum:
```suggestion
    ssoType: 'OIDC' | 'SAML';
```
But at the end of the day, it's only being used as a string for now anyways, so 🤷‍♂️